CMakeLists.txt

```diff
@@ -112,8 +112,12 @@ add_subdirectory(tutorials)
 set(core_src "${cinnapi_src}")
 
 set(Python_VIRTUALENV FIRST)
+if(NOT PYTHON_EXECUTABLE)
 find_package(PythonInterp ${PY_VERSION} REQUIRED)
```

Comment:
Add space at the beginning of line 116 and line 119?

Reply:
I will refine it in the next pr.